docs/conf.py

```diff
@@ -248,8 +248,9 @@ def generate_r_docs(app):
     source /home/docs/.conda/bin/activate r_env
     export TAR=/bin/tar
     cd {0}
+    git submodule update --init --recursive
     export R_LIBS="$CONDA_PREFIX/lib/R/library"
-    Rscript build_r.R
+    Rscript build_r.R || exit 1
```

Comment:
Please use `-1` for the consistency. It may help to simply a search query in the repo if we decide to change something in the future.

Shouldn't `Rscript -e "pkgdown::build_site()` cause builds to fail as well?

Reply:
yeah you're right